parasol_cpu/src/proc/assembly.rs

```diff
@@ -8,10 +8,11 @@ macro_rules! define_op {
         $dispatch_name:ident,
         ($($reg_kind:ty),*),
```

Comment:
Include the register count here somehow so we can derive the mask and shifts below

Reply:
Another review.